src/k2/cmd/httpclient/http_client.cpp

```diff
@@ -162,8 +162,21 @@ class HTTPClient {
         if (myid == 0) {
             K2LOG_I(k2::log::httpclient, "Creating collection...");
             _startFut = _startFut.then([this] {
-                return _client.makeCollection(collname).discard_result()
-                .then([this] () {
+                k2::dto::CollectionMetadata meta {
```

Comment:
I think I missed this before - what is this collection we're creating in the HTTP client?